src/main/java/org/broadinstitute/hellbender/tools/walkers/gnarlyGenotyper/GnarlyGenotyper.java

```diff
@@ -173,8 +173,9 @@ public boolean requiresReference() {
     @Override
     protected GenomicsDBOptions getGenomicsDBOptions() {
         if (genomicsDBOptions == null) {
-            genomicsdbArgs.maxDiploidAltAllelesThatCanBeGenotyped = PIPELINE_MAX_ALT_COUNT;
-            genomicsDBOptions = new GenomicsDBOptions(referenceArguments.getReferencePath(), genomicsdbArgs, genotypeArgs, CALL_GENOTYPES);
+            genomicsdbArgs.callGenotypes = CALL_GENOTYPES;
```

Comment:
I think this is a minor bug.  Previously I think it would force call if either `--call-genotypes` was set OR `CALL_GENOTYPES` was true.  Now it only takes `CALL_GENOTYPES` into account.

Reply:
Since [CALL_GENOTYPES](https://github.com/broadinstitute/gatk/blob/3924f9a9833afd57ece41f6c0d9207794fd19268/src/main/java/org/broadinstitute/hellbender/tools/walkers/gnarlyGenotyper/GnarlyGenotyper.java#L98) is hardcoded to true, I wanted to explicitly override the arguments and be consistent with what is done for  [maxDiploidAltAllelesThatCanBeGenotyped](https://github.com/broadinstitute/gatk/blob/3924f9a9833afd57ece41f6c0d9207794fd19268/src/main/java/org/broadinstitute/hellbender/tools/walkers/gnarlyGenotyper/GnarlyGenotyper.java#L176).